spec/system/page_objects/pages/user.rb

```diff
@@ -24,11 +24,32 @@ def has_warning_messages_path?(user)
         page.has_current_path?("/u/#{user.username}/messages/warnings")
       end
 
+      def staff_info_section
+        begin
+          page.find(".staff-counters")
```

Comment:
note: I initially wanted to go around the need to rescue from the Capybara error by using the `first`/`all` finder methods, but those had flaky results across test runs / re-builds of the frontend assets. `find` is more consistent for some reason.